addons/google-calendar/package.json

```diff
@@ -3,6 +3,6 @@
     "node": "10"
   },
   "dependencies": {
-      "@fusebit/add-on-sdk": "^0.0.1"
+    "@fusebit/add-on-sdk": "^0.0.1"
```

Comment:
Update sdk version?

Reply:
Won't fix. This PR is scoped to Slack, I'd rather not touch other addons in it... all I did was prettify the code